ec_api/apps/api_docs/blueprints/data_structures.apibp

```diff
@@ -114,18 +120,6 @@
                         + person: (object)
                             + `ynr_id`: `23417` (number)
                             + name: `David Keigwin` (string)
-                            + `email`: `dave@example.com` (string, nullable) - Email address for this candidate, if we hold it
-                            + `photo_url`: (string, nullable) - URL for a photo of this candidate, if we hold one
```

Comment:
There's another small thing I did here. These properties get stripped off in this API so should not appear in the docs

https://github.com/DemocracyClub/ec-api-proxy/blob/5f7141343e11b14cd8eeee73b64d4e4d64cf7560/api_endpoints/v1_postcode_lookup/upstream_api_client.py#L50-L56